time    tick      # Time
float64 mag_x     # Body-frame X magnetic field strength (Tesla)
float64 mag_y     # Body-frame Y magnetic field strength (Tesla)
float64 mag_z     # Body-frame Z magnetic field strength (Tesla)